packages/server/package.json

```diff
@@ -2,16 +2,56 @@
   "name": "@online-games-shop/server",
   "version": "1.0.0",
   "scripts": {
-    "watch-ts": "tsc -w",
+    "watch": "tsc -w",
```

Comment:
I suggest you to use `ts-node` + `nodemon` for such watch & compile stuff.

Example: https://github.com/max-starikevich/lasertag-scheduler-bot/blob/master/nodemon.json